examples/CV7/CV7_example.c

```diff
@@ -73,12 +73,13 @@ const uint8_t FILTER_PITCH_EVENT_ACTION_ID = 2;
 //Required MIP interface user-defined functions
 timestamp_type get_current_timestamp();
 
-bool mip_interface_user_recv_from_device(mip_interface* device, uint8_t* buffer, size_t max_length, size_t* out_length, timestamp_type* timestamp_out);
```

Comment:
Personally I don't like this style of line continuation. It looks ugly when there are lots of functions with different indentation levels (due to the names being different lengths). Plus for really long function names there isn't much space for the parameters to begin with.

I'd prefer starting the parameters on a new indented line, like this:
```
bool mip_interface_user_recv_from_device(
  mip_interface* device, uint8_t* buffer, size_t max_length, size_t* out_length, timestamp_type* timestamp_out
);
```
or
```
bool mip_interface_user_recv_from_device(
  mip_interface* device, uint8_t* buffer, size_t max_length,
  size_t* out_length, timestamp_type* timestamp_out
);
```